src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpOnFilenameCheck.java

```diff
@@ -121,6 +128,13 @@
  * &lt;/module&gt;
  * </pre>
  *
+ * <p>Example:</p>
```

Comment:
please move this under config with `fileNamePattern`.

Please change config above this Example to cover use case "No '*.md' files other then 'README.md'", please google on how to make such regexp.
Please use property `message` in such example to show users that they can customize message.

Reply:
@romani I will also add appropriate example below this new config.

Reply:
Done